code/materials/_material.dm

```diff
@@ -40,6 +40,16 @@
 	var/list/crafting_recipe_paths = list()
 	var/list/recipes = list()
 
+	// Turf vars.
+	var/turf_effect_overlay
+	var/turf_floor_icon
+	var/turf_wall_icon
+	var/turf_is_diggable
+	var/turf_edge_layer
+	var/turf_base_states = 1
+	var/turf_wall_is_dense = 1
+	var/turf_wall_is_transparent = 0
```

Comment:
I know I need to start doing it more in my own code as well, but is it possible to get brief comments on what each of these vars are for?